tests/core/a-entity.test.js

```diff
@@ -1045,6 +1045,19 @@ suite('a-entity', function () {
       assert.equal(el.getAttribute('sound__2').src, soundUrl);
       assert.equal(el.getAttribute('sound__2').autoplay, true);
     });
+
+    test('clear mixin', function () {
+      var el = this.el;
+      mixinFactory('material', {material: 'shader: flat'});
+      mixinFactory('position', {position: '1 2 3'});
+      el.setAttribute('mixin', 'material position');
```

Comment:
can we check for `el.mixinEls` too? `assert.equal(el.mixinEls.length, 2)` before and `assert.equal(el.mixinEls.length, 0)` after